{
  "id": "EUVD-2025-32691",
  "enisaUuid": "7c63c726-2bba-3e7a-bfe7-6ff54831eb5b",
  "description": "Django vulnerable to SQL injection in column aliases",
  "datePublished": "Oct 1, 2025, 9:31:21 PM",
  "dateUpdated": "Oct 13, 2025, 3:14:13 PM",
  "baseScore": 7.1,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:C/C:H/I:L/A:N",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2025-59681",
    "https://github.com/django/django/commit/41b43c74bda19753c757036673ea9db74acf494a",
    "https://github.com/django/django/commit/43d84aef04a9e71164c21a74885996981857e66e",
    "https://docs.djangoproject.com/en/dev/releases/security",
    "https://github.com/django/django",
    "https://groups.google.com/g/django-announce",
    "https://www.djangoproject.com/weblog/2025/oct/01/security-releases",
    "https://docs.djangoproject.com/en/dev/releases/security/",
    "https://www.djangoproject.com/weblog/2025/oct/01/security-releases/"
  ],
  "aliases": [
    "CVE-2025-59681",
    "GHSA-hpr9-3m2g-3j9p"
  ],
  "assigner": "mitre",
  "epss": 0.02,
  "enisaIdProduct": [
    {
      "id": "225b043c-bcbd-3b12-b8ec-4294f29b0857",
      "product": {
        "name": "Django"
      },
      "product_version": "4.2 <4.2.25"
    },
    {
      "id": "bb033068-48f1-33ff-9b2d-91e278b1642d",
      "product": {
        "name": "Django"
      },
      "product_version": "5.1 <5.1.13"
    },
    {
      "id": "be800cd6-ce08-3752-b33d-dc6b0e49d4f6",
      "product": {
        "name": "Django"
      },
      "product_version": "5.2 <5.2.7"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "a68f82c1-4299-3e99-9f67-611ede909890",
      "vendor": {
        "name": "djangoproject"
      }
    }
  ]
}